packages/trpc/server/routers/viewer/availability/schedule/getScheduleByUserId.handler.ts

```diff
@@ -0,0 +1,56 @@
+import { DEFAULT_SCHEDULE } from "@calcom/lib/availability";
+
+import { TRPCError } from "@trpc/server";
+
+import type { TrpcSessionUser } from "../../../../trpc";
+import { getHandler } from "./get.handler";
+import type { TGetByUserIdInputSchema } from "./getScheduleByUserId.schema";
+
+type GetOptions = {
+  ctx: {
+    user: NonNullable<TrpcSessionUser>;
+  };
+  input: TGetByUserIdInputSchema;
+};
+
+export const getScheduleByUserIdHandler = async ({ ctx, input }: GetOptions) => {
```

Comment:
This is used as a wrapper for the get.handler function -> since we only have UID we use this to get the default schedule id from this then call that handler